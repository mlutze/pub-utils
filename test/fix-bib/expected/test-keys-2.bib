@article{madsen2022,
  title   = {Polymorphic types and effects with Boolean unification},
  author  = {Magnus Madsen and Jaco van de Pol},
  journal = {Proceedings of the {ACM} on Programming Languages},
  year    = {2020},
  doi     = {10.1145/3428222}
}

@article{madsen2020b,
  title   = {Another Publication},
  author  = {Magnus Madsen and Jaco van de Pol},
  journal = {Proceedings of the {ACM} on Programming Languages},
  year    = {2020},
  doi     = {1234567}
}

