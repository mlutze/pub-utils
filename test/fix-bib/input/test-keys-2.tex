\documentclass{article}

\begin{document}
\cite{somename}
\cite{othername}
\end{document}