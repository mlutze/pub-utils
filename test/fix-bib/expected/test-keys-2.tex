\documentclass{article}

\begin{document}
\cite{madsen2020}
\cite{madsen2020b}
\end{document}